### PR TITLE
BGDIDIC-1780: Add/update ch.bak.bundesinventar-schuetzenswerte-ortsbilder*

### PR DESCRIPTION
**Pull request by b4l**
Test link: https://mf-chsdi3.dev.bgdi.ch/feature_BGDIDIC-1780/shorten/8f9533d113

Mapfile: https://github.com/geoadmin/wms-mapfile_include/pull/628

Script: https://github.com/geoadmin/bgdi-scripts/pull/848

Known bugs:
- [x] Foto direction indicator does not preserve glyph translations.
- [x] Perimeter on zoomlevel 6 is not recognized as identifiable geom.
- [x] Tooltip <> geometry display mismatch on some zoomlevel

### CODE REVIEW

File: chsdi/models/vector/bak.py [hunk 1]
```diff
@@ -10,30 +10,84 @@
...
+
+class IsosOrtsbildPerimeter(Base, IsosBase, Vector):
+    __tablename__ = 'view_isos_ortsbild'
+    __bodId__ = 'ch.bak.bundesinventar-schuetzenswerte-ortsbilder'
+    __label__ = 'name'
+    __queryable_attributes__ = ['id', 'name']
+    __returnedGeometry__ = 'perimeter'
+    __maxscale__ = 50000
+    __minscale__ = 25000
+    perimeter = Column('perimeter', Geometry2D)
```
Comment (b4l) [inline chsdi/models/vector/bak.py:41]: Somehow `the_geom = Column('perimeter', Geometry2D)` throws an error even though there are some working examples.
Also this (polygon) geometry is not recognized as a clickable area to identify, only some point behind.

Comment (rebert) [reply chsdi/models/vector/bak.py:41]: I don't know - what happens when the attribute always is called the_geom?

the_geom = Column('perimeter', Geometry2D)

Comment (b4l) [reply chsdi/models/vector/bak.py:41]: As mentioned it throws an error upon deploying mf-chsdi3.

Comment (ltclm) [reply chsdi/models/vector/bak.py:41]: did you try to make two views instead with spatial column the_geom?

Comment (ltclm) [reply chsdi/models/vector/bak.py:41]: whats the error message?

**Review by rebert — APPROVED** [review]
Good job! Beer!!!!

**Pull request merged**